website/content/en/docs/building-operators/helm/quickstart.md

```diff
@@ -5,326 +5,88 @@ weight: 100
 description: A simple set of instructions that demonstrates the basics of setting up and running a Helm-based operator.
 ---
 
-This guide walks through an example of building a simple nginx-operator powered by [Helm][helm-official] using tools and libraries provided by the Operator SDK.
+This guide walks through an example of building a simple nginx operator
+powered by Helm using tools and libraries provided by the Operator SDK.
```

Comment:
We should add the official Helm link back. I may have accidentally dropped that in the commit that I pushed.